src/core_landice/Registry.xml

```diff
@@ -165,6 +165,42 @@
                         description="If true, then ice from small grounded islands is removed.  Specifically, this finds one- and two-cell masses of ice that are surrounded by open ocean and eliminates them by sending them to the calving flux.  Grounded such ice masses would be islands, floating would be icebergs.  The ice removed is added to the calvingThickness field."
 	                possible_values=".true. or .false."
 		/>
+		<nml_option name="config_restore_damage" type="logical" default_value=".false." units="unitless"
+                        description="If true, then turn on the restore_damage function."
+		            possible_values=".true. or .false."
+                />
+		<nml_option name="config_calculate_damage" type="logical" default_value=".false." units="unitless"
+                        description="If true, then turn on the calculate_damage function."
+		            possible_values=".true. or .false."
+                />
+                <nml_option name="config_damage_restore_threshold" type="real" default_value="0.0" units="unitless"
+                        description="It is a value above which the old damage number would be kept from the impact of advection (the damage would not heal). For example, if the damage at some location is 0.83, and we set config_damage_restore_threshold to 0.8, then the damage number there will not fall below 0.83. Otherwise, if the damage is 0.7, then it could be decreased to 0.6 if non-damaged ice is transported from upstream. If we set config_damage_restore_threshold to 0 (default), then it will just not play a role in pratical."
```

Comment:
I interpret this as allowing you to decide if you want damage healing on or off, depending on the value of the damage (e.g., let it heal below some threshold value but not heal above that same value). I can imagine this is useful for testing but I'm having trouble coming up with a specific example offhand.

Reply:
Steve did a very nice explanation. It was also sort of an idea from Jeremy's discussion before, that the ice shelf will not heal if it is damaged over a threshold. But like Steve, I don't know a specific example either.

Reply:
Thanks for that reminder Tong. We did indeed have a discussion where Jeremy suggested that this might be something we'd need to add / look into at some point if we were hoping to get damage / calving front positions correct for real ice shelves. Tong, maybe you can clarify if we are using this at all right now or not (e.g., in any of your existing test cases). I think it's fine to leave that functionality in if it is already working in the code. Maybe we just need a bit more of a clear description for why this functionality exists.

Reply:
Not for current examples. This can be realized by setting config_damage_restore_threshold, say, if we set it to 0.8, then only the locations with damage > 0.8 can not be healded. In the current simulations, I set config_damage_restore_threshold = 0, which means the ice shelf can't be healed everywhere. I updated the description a bit.